gnomad_qc/v4/resources/basics.py

```diff
@@ -104,16 +104,13 @@ def _remove_ukb_dup_by_index(
 
     logger.info("Total number of UKB samples to exclude: %d", len(withdrawn_ids))
 
-    with hl.hadoop_open(all_ukb_samples_to_remove, "w") as d:
-        for sample in withdrawn_ids:
-            d.write(sample + "\n")
-
-    withdrawn_ht = hl.import_table(all_ukb_samples_to_remove, no_header=True).key_by(
-        "f0"
+    vds = hl.vds.filter_samples(
+        vds, 
+        withdrawn_ids, 
+        keep=False, 
+        remove_dead_alleles=True,
     )
```

Comment:
Trying again to get black format correct
```suggestion
    vds = hl.vds.filter_samples(
        vds,
        withdrawn_ids,
        keep=False,
        remove_dead_alleles=True,
    )
```